lib/helper.js

```diff
@@ -170,10 +175,68 @@ async function deleteAllLabels(token, owner, host, repository, cli) {
 
     // Done
     if (cli.flags.uploadLabels) echo.success('Done!\n')
-    else {
+    else if (exit) {
         console.log()
-        echo.success('Finished!', true)
+        echo.success('Finished!', exit)
+    }
+}
+
+// Gets array of repository names in "owner" organization
+async function getRepositories(token, owner, host, cli) {
```

Comment:
The parameter `cli` is not used so there's no need to receive it.
```suggestion
async function getRepositories(token, owner, host) {
```